recipes/install_from_source.rb

```diff
@@ -0,0 +1,65 @@
+#
+# Cookbook Name:: qubit-bamboo
+# Recipe:: default
+#
+# Copyright (c) 2015 The Authors, All Rights Reserved.
+#
+
+include_recipe 'yum-epel'
+
+package 'golang'
+package 'git'
+
+tar = ::File.join(Chef::Config[:file_cache_path], "qubit_bamboo_#{node['qubit_bamboo']['version']}.tar.gz")
+remote_file tar do
+  source "https://github.com/QubitProducts/bamboo/archive/v#{node['qubit_bamboo']['version']}.tar.gz"
+end
+
+bash 'untar_bamboo' do
+  cwd "#{Chef::Config[:file_cache_path]}/"
+  code <<-EOF
+  tar xzf #{tar}
+  EOF
+end
+
+directory node['qubit_bamboo']['home'] do
+  recursive true
+end
+
+gopath = ::File.join(Chef::Config[:file_cache_path], 'go')
+go_link = ::File.join(gopath, 'src', 'github.com', 'QubitProducts')
+directory go_link do
+  recursive true
+end
+
+bash 'build_bamboo' do
```

Comment:
This recipe will build from source and copy over the new binary every single time. We should only build once.